onebusaway-android/src/main/java/org/onebusaway/android/util/UIUtils.java

```diff
@@ -1141,4 +1138,19 @@ public static String formatTime(Context context, long time) {
                         DateUtils.FORMAT_NO_MIDNIGHT
         );
     }
+
+    /**
+     * Set smaller text size if the route short name has more than 3 characters
+     *
+     * @param view Text view
+     * @param routeShortName Route short name
+     */
+    public static void setTextSizeForArrivalsListAdapterStyleARoute(TextView view,
+                                                                    String routeShortName) {
+        if (routeShortName.length() == 4) {
+            view.setTextSize(30);
```

Comment:
Could you please move the `30` and `24` numbers here to `dimens.xml`?  Check out http://stackoverflow.com/a/16718062/937715 for how to read this correctly.